common/src/main/resources/Operator.properties

```diff
@@ -59,11 +59,11 @@ WLSKO-0087=Waiting for pod {0} to be ready
 WLSKO-0088=Pod {0} is ready
 WLSKO-0089=Pod {0} has failed
 WLSKO-0094=Async call {0} invoking: {1}, namespace: {2}, name: {3}, body: {4}, fieldSelector: {5}, labelSelector: {6}, resourceVersion: {7}
-WLSKO-0095=Async call {0} invoking: {1}, failed: {2}, code: {3}, headers {4}, namespace: {5}, name: {6}, body: {7}, fieldSelector: {8}, labelSelector: {9}, resourceVersion: {10}, response body: {11}
```

Comment:
You've accidentally reverted changes by others. I'll try and catch them but please go through and look for bad merges.

Reply:
Fixed